cypress/views/mcg-ms-common.ts

```diff
@@ -0,0 +1,22 @@
+import { DATA_FEDERATION, DATA_SERVICES, NO_WAIT } from '../constants/common';
+
+export const MCG_MS_Common = {
+  visitMcgMsDashboard: () => {
+    cy.get('[data-test="nav"]', { timeout: 15 * 1000 })
+      .should('contain', DATA_SERVICES)
+      .within(() => {
+        cy.contains(DATA_SERVICES, NO_WAIT)
+          .should('have.attr', 'aria-expanded', 'false')
```

Comment:
why not chain it with first should itself... or some other way...
contains inside another contains and both check same thing... not looks good way to chain things...

Reply:
we can use then approach hereafter querying the particular HTML element after the first should

Reply:
we can either use within or then to keep the context within the selected HTML element